### ISSUE TITLE
Update Toil version to include latest cactus fixes.

### ISSUE THREAD

**Issue by DailyDreaming** [ComparativeGenomicsToolkit/cactus#395]
The most recent version I've pushed today (5.2.0) should have fixes for s3 and a weird recursion bug that was hit:

https://github.com/DataBiosphere/toil/issues/3366
https://github.com/DataBiosphere/toil/issues/3367
https://github.com/DataBiosphere/toil/issues/3393

But cactus seems to be much more out of date than this (4.2.0) and would benefit from updating.  Especially since I don't want people to come by with cactus bugs that have already been solved because cactus is out of date.

**Comment by glennhickey** [comment]
I agree!  The reason I don't update to each Toil release is that it [breaks the aws/mesos workflow pretty much every time](https://github.com/DataBiosphere/toil/issues/3414).  And since Cactus doesn't have an automated test for that, it takes a lot of manual checking and back-and-forth.  Changes to the Toil CLI also necessitate updates to the Cactus documentation.   

If we could somehow get a test into Cactus or Toil that verifies the procedure outlined [here](https://github.com/DataBiosphere/toil/issues/3034) I think it would help to keep things in sync.

**Comment by glennhickey** [comment]
[The latest Toil version also seems about 25X slower at importing S3 files into an S3 jobstore](https://github.com/DataBiosphere/toil/issues/3366#issuecomment-766104628).   For the 92 sample pangenome workflow, this would add up to nearly a 30% wall-time increase. 

Am also concerned about the disappearance of the `--defaultPreemptable` option, but haven't had time to test that yet. .
